### DISCUSSION TITLE
Can we add this? https://github.com/haoheliu/AudioLDM

### DISCUSSION THREAD

**Discussion by vdruts** [vladmandic/sdnext#226] [Ideas]
Check this out, would be awesome https://github.com/haoheliu/AudioLDM

**Comment by vladmandic** [comment]
it does look cool, but how is that related to txt2img, seems like a completely separate project.

**Reply by ke1ne** [reply]
It seems it can be implemented as extension with a separate tab. It has ckpts and code example on Huggingface.

**Comment by vdruts** [comment]
Well it's text to (sound) which can easily augment creative projects done with text-image/video , especially with all of the plugins for auto that do that (Sadtalker) etc

**Reply by ke1ne** [reply]
If you have proper skills, implement it as an extension with a separate tab, like Deforum, etc. I think it could be a good compromise, instead of a core integration or something.

**Comment by ke1ne** [comment]
@bbecausereasonss Maybe, I did lost something, but where are checkpoint files to use with their code?
UPD. Nevermind, found them https://huggingface.co/spaces/haoheliu/audioldm-text-to-audio-generation/tree/main
